plugins/remark-lint-no-dead-urls/src/lib.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import got from 'got';
+
+export async function fetch(url, method, options = {}) {
+  const {statusCode} = await got(url, {
```

Comment:
We should add a user-agent for this, to reduce the chance of being rate-limited.

Reply:
<img width="305" alt="CleanShot 2023-02-23 at 09 35 21@2x" src="https://user-images.githubusercontent.com/49578/220869992-486d4638-c1c7-447a-a598-f16a2bd31ac2.png">